lib/theme/widgets/hs_warning_bottom_sheet.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class HsWarningBottomSheetView extends StatelessWidget {
+  // final ValueChanged value;
+  final String? iconUrl;
+  final String title;
+  final String description;
+  final bool enablePrimaryButton;
+  final String primaryButtonLabel; // return false value
+  final VoidCallback? primaryOnPressed;
+  final bool enableSecondaryButton;
+  final String secondaryButtonLabel; // return true value
+  final VoidCallback? secondaryOnPressed;
+  const HsWarningBottomSheetView(
+      {super.key,
+      // required this.value,
+      this.iconUrl,
+      String? title,
+      String? description,
+      String? primaryButtonLabel,
+      bool? enablePrimaryButton,
+      this.primaryOnPressed,
+      bool? enableSecondaryButton,
+      String? secondaryButtonLabel,
+      this.secondaryOnPressed})
+      : title = title ?? "Modal title",
+        description = description ?? "Modal description",
+        enablePrimaryButton = enablePrimaryButton ?? false,
+        primaryButtonLabel = primaryButtonLabel ?? "",
+        enableSecondaryButton = enableSecondaryButton ?? false,
+        secondaryButtonLabel = secondaryButtonLabel ?? "";
+  @override
+  Widget build(BuildContext context) {
+    List<Widget> modalContent = [];
+    List<Widget> buttonGroup = [];
+    Widget button;
+    // TODO: implement build
+    if (iconUrl == null) {
```

Comment:
Don't do this.
Check if iconUrl != null -> add Icon
if (title != null) -> add Title
if (description != null) -> add Description

-> will make your code more flexible